snuba/datasets/storages/errors.py

```diff
@@ -28,6 +30,20 @@
 from snuba.datasets.table_storage import KafkaStreamLoader
 from snuba.query.processors.prewhere import PrewhereProcessor
 
+
+def errors_migrations(
+    clickhouse_table: str, current_schema: Mapping[str, ColumnType]
+) -> Sequence[str]:
+    ret = []
+
+    if "message_timestamp" not in current_schema:
+        ret.append(
+            f"ALTER TABLE {clickhouse_table} ADD COLUMN message_timestamp DateTime AFTER offset"
+        )
+
+    return ret
```

Comment:
I don't know if this is required or not, so I added it to be safe. It's easy to back out if not required.

I think the default value is fine here.

Reply:
It is required indeed for developers and onpremise.